python/lsst/summit/utils/utils.py

```diff
@@ -1203,3 +1207,56 @@ def computeCcdExposureId(instrument: str, exposureId: int, detector: int) -> int
         return lsst.obs.lsst.translators.LsstCamTranslator.compute_detector_exposure_id(exposureId, detector)
     else:
         raise ValueError("Unknown instrument {instrument}")
+
+
+def getDetectorIds(instrumentName: str) -> list[int]:
+    """Get a list of detector IDs for a given instrument.
+
+    Parameters
+    ----------
+    instrumentName : `str`
+        The name of the instrument.
+
+    Returns
+    -------
+    detectorIds : `list` of `int`
+        The list of detector IDs.
+    """
+    camera = getCameraFromInstrumentName(instrumentName)
+    return [detector.getId() for detector in camera]
+
+
+def getCameraFromInstrumentName(instrumentName: str) -> lsst.afw.cameraGeom.Camera:
+    """Get the camera object given the instrument name (case insenstive).
+
+    Parameters
+    ----------
+    instrumentName : `str`
+        The name of the instrument, e.g. "LATISS" or "LSSTCam". Case
+        insenstive.
```

Comment:
So these are the same names as used in the butler data IDs, right?  You're just being more permissive about case?

Reply:
Yes, exactly. ConsDB is all lower case, so it's now quite useful to do so.